spring-cloud-generator/generate-one.sh

```diff
@@ -73,40 +73,46 @@ bazel build //$googleapis_folder:"$client_lib_name"_java_gapic_spring
 cd -
 
 ## copy spring code to outside
-mkdir -p ../generated
-cp googleapis/bazel-bin/$googleapis_folder/"$client_lib_name"_java_gapic_spring-spring.srcjar ../generated
+mkdir -p ../spring-cloud-previews
```

Comment:
Nit: could we maybe use an exit code to condition these post-process / add to pom steps on the success status of the bazel build call above?

Reply:
Added exit on error option to address this for now ([commit](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1394/commits/6af2bdf772b6bcd8df56368ec72915d07a1fdddb)). 
I would keep track of a separate task to improve error handling overall.